### PR TITLE
feat(dashboard): Deploys UI

### PR DESCRIPTION
**Pull request by lynnagara**

### CODE REVIEW

**Review by ckj — APPROVED** [review]
Nice! Looks good with deploy activit and not so good without it. I think that's something we should iterate at some point after this lands.

File: src/sentry/static/sentry/app/views/organizationDashboard/deploys.jsx [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+                key={deploy.version}
+                deploy={deploy}
+                projectId={projectId}
+                orgId={orgId}
+              />
+            ))}
+          </div>
+        </DeployBox>
+      );
+    } else {
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationDashboard/deploys.jsx]: This `else` is un-needed.

Just a style opinion, I think it's easier to read if the "main" component is the "default" condition and placed at the end. e.g.

```
if (cond) return <NoDeploys />;

return <MainStuff />;
```

Thoughts?

File: src/sentry/static/sentry/app/views/organizationDashboard/deploys.jsx [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+const Heading = styled.div`
+  color: ${p => p.theme.gray2};
+  text-transform: uppercase;
+  font-size: 14px;
+`;
+
+class Deploy extends React.Component {
+  static propTypes = {
+    deploy: PropTypes.object.isRequired,
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationDashboard/deploys.jsx]: Can we describe the `shape` of `deploy` here in `SentryTypes`?

File: src/sentry/static/sentry/app/views/organizationDashboard/deploys.jsx [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+class NoDeploys extends React.Component {
+  render() {
+    return (
+      <DeployBox p={2} pb={0}>
+        <Background align="center" justify="center">
+          <Button
+            size="xsmall"
+            href="https://blog.sentry.io/2017/05/09/release-deploys"
+            target="_blank"
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationDashboard/deploys.jsx]: If you use `external: true`, it will use the `ExternalLink` component

**Review by billyvg — COMMENTED** [review]
It'd be good if we had some mocked deploy data in our acceptance tests.

Also we may want to update the team section placeholder height for lazy load.

File: src/sentry/static/sentry/app/views/organizationDashboard/deploys.jsx (hunk 1 above)
Comment (lynnagara) [reply src/sentry/static/sentry/app/views/organizationDashboard/deploys.jsx]: Yeah, I've seen this style often in our codebase. I like symmetric if/else better though.

**Pull request merged**